cypress/support/index.js

```diff
@@ -1,6 +1,33 @@
+beforeEach(() => {
+  cy.fixture('user_config').as('userConfig');
+  cy.server();
+  cy.route('/user-config/config.json', '@userConfig');
+});
+
 Cypress.Commands.add('login', () => {
-  cy.fixture('realm').then((realm) => {
-    cy.visit(`/auth?realm=${realm.name}#access_token=${realm.infinite_token}`);
-    cy.wait(500);
-  });
+  cy.fixture('user_config')
+    .as('userConfig')
+    .then((userConfig) => {
+      cy.server();
+      cy.route('/user-config/config.json', '@userConfig').as('getUserConfig');
+      cy.fixture('realm').then((realm) => {
+        const apiUrl = new URL(userConfig.astarte_api_url).hostname;
+        const session = {
+          login_type: 'TokenLogin',
+          api_config: {
+            secure_connection: true,
+            realm_management_url: `${apiUrl}/realmmanagement`,
+            appengine_url: `${apiUrl}/appengine`,
+            pairing_url: `${apiUrl}/pairing`,
+            flow_url: `${apiUrl}/flow`,
+            realm: realm.name,
+            token: realm.infinite_token,
+            enable_flow_preview: userConfig.enable_flow_preview,
+          },
+        };
+        localStorage.session = JSON.stringify(session);
+        cy.visit('/');
```

Comment:
no need to visit the home page. Actually this makes the login process even longer